tools/zip_examples.py

```diff
@@ -0,0 +1,86 @@
+# !python
+
+import argparse
+import logging
+import os
+import pprint
+import shutil
+import sys
+import tempfile
+import time
+import zipfile
+
```

Comment:
I think it makes sense to have top level documentation of what this script does, because to the regular viewer this seems overly convoluted.

It should be displayed when the script is invoked with `--help`.

Suggested:

```
Script for compressing a directory of Python examples as a ZIP file in a way that prevents the resulting file from being binary different if the source files didn't change.
* Reset the file modification timestamps on each file because these are preserved during zipping.
* Ignore .pyc files that may be encountered.
```

Reply:
Changed